src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -92,6 +92,26 @@ public static FunctionPointerMethodSymbol CreateFromSource(FunctionPointerTypeSy
                 suppressUseSiteDiagnostics);
         }
 
+        /// <summary>
+        /// Creates a function pointer method symbol from individual parts. This method should only be used when diagnostics are not needed.
+        /// </summary>
+        internal static FunctionPointerMethodSymbol CreateFromParts(
+            TypeWithAnnotations returnType,
+            RefKind returnRefKind,
+            ImmutableArray<TypeWithAnnotations> parameterTypes,
+            ImmutableArray<RefKind> parameterRefKinds,
+            CSharpCompilation compilation)
+        {
+            return new FunctionPointerMethodSymbol(
+                // https://github.com/dotnet/roslyn/issues/39865 allow setting this in creation
```

Comment:
>// https://github.com/dotnet/roslyn/issues/39865 allow setting this in creation [](start = 16, length = 79)

Does this refer to allowing `CallingConvention` to be settable? If so, does the comment belong on the public API instead?